Objects/typeobject.c

```diff
@@ -8795,7 +8795,7 @@ wrap_indexargfunc(PyObject *self, PyObject *args, void *wrapped)
     PyObject* o;
     Py_ssize_t i;
 
-    if (!PyArg_UnpackTuple(args, "", 1, 1, &o))
+    if (!PyArg_UnpackTuple(args, "__mul__", 1, 1, &o))
```

Comment:
This code is called for `__mul__`, `__rmul__` and `__imul__`.

I think that it is easier to use `check_num_args()` here.